tasks-by-users/Naufil-Shemle/03-css-positioning/style.css

```diff
@@ -0,0 +1,148 @@
+body {
```

Comment:
It's not a good practise to give properties to body, instead create another class and wrap these properties in it.

Reply:
As you can you see brother my stuff is in a class but that class size is less than window size that's why I used this.
And I wanted to show the window color and my class color differently

Reply:
its okay for now but take account of it in future

Reply:
Inshaallah